libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -265,6 +265,13 @@ const AP_Param::GroupInfo RC_Channel::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("OPTION",  6, RC_Channel, option, 0),
 
+    // @Param: OPTIONS
```

Comment:
please change this name! surely we can find a non-confusing one.... perhaps
```suggestion
    // @Param: SW_BEHAVIOR
```

Reply:
Can we rename RCx_OPTION to RCxx_FUNCTION and use RCxx_OPTIONS as a new parameter for behaviour and possibly other options related to function input?

Reply:
> please change this name! surely we can find a non-confusing one.... perhaps

Well, this is a generic options bitmask, so `SW_BEHAVIOUR`  isn't right.

The reason there's a "TBA" bit in here is that we might want to move the "reversed" flag into here.  Its a possibility, but I'm a little doubtful we will move it

Definitely open to suggestions here, because I do realise that we can't *really* do `_OPTIONS` :-)

We could rename the existing thingy to `RC1_FUNCTION`,for example.  We did actually use the word "function" way, way back when, I think I changed it to OPTION when I moved things into RC_Channel.

Could go for `RC1_OPTS` as different-enough?

Reply:
> Can we rename RCx_OPTION to RCxx_FUNCTION and use RCxx_OPTIONS as a new parameter for behaviour and possibly other options related to function input?

That's going to make for awkward documentation and confusion between version.  Perhaps we can do that but restrict ourselves to `RC1_OPTS` instead....

Reply:
The best answer would be to change the existing _OPTIONS to _FUNCTION. Even now I find it confusing that it's SERVOx_FUNCTION but RCx_OPTION - it would be great to fix that and IMO makes the whole system less confusing than it is now. 

> Oh no! A learning experience! The horror! :-)

IMO it's totally worth it, but if you really don't want to do it then I don't see how you can argue that _BEHAVIOR "just isn't right". What we have now isn't right, _BEHAVIOR is less worse than that.

Reply:
Can we use two digit numbers if we are changing names, please?
RC01_...
RC02_...
...
RC11_...

Reply:
> Can we use two digit numbers if we are changing names, please? RC01_... RC02_... ... RC11_...

This scheme was flat-out rejected when I suggested it for `MAV01`, `MAV02`, `MAV03`.

I have no reason to believe any suggestion to do it here would have some other result.